internal/impl/sql/output_sql_insert.go

```diff
@@ -148,6 +149,16 @@ func newSQLInsertOutputFromConfig(conf *service.ParsedConfig, logger *service.Lo
 	s.builder = squirrel.Insert(tableStr).Columns(columns...)
 	if s.driver == "postgres" || s.driver == "clickhouse" {
 		s.builder = s.builder.PlaceholderFormat(squirrel.Dollar)
+	} else if s.driver == "oracle" {
+		s.builder = s.builder.PlaceholderFormat(squirrel.Colon)
+	}
+
+	if s.useTxStmt {
+		values := make([]any, 0, len(columns))
```

Comment:
Kludge: `Values` requires `...interface{}` and `columns` is `[]string` so I can't just use `columns...`